src/src/components/Login.vue

```diff
@@ -0,0 +1,126 @@
+<template>
+  <form class="login" @submit.prevent="handleLogin">
+    <img src="../assets/images/vaultN-logo.svg" class="login-logo" />
+    <h1>{{ contentLogin.loginHeadlineText }}</h1>
+    <CustomInput :input="customEmailInput" @valueChanged="handleTextChanged"/>
+    <CustomInput :input="customPasswordInput" @valueChanged="handleTextChanged"/>
+    <FormErrors v-if="errroMessages" :messages="errroMessages"></FormErrors>
+    <a href="#/forgot" class="link">{{ contentLogin.loginForgotPasswordText }}</a>
+    <CustomButton :button="customLoginButton" @buttonClicked="handleLogin"/>
+    <a href="#/reset" class="link">{{ contentLogin.loginCreateAccountText }}</a>
+  </form>
+</template>
+
+<script lang="ts">
+import {
+  computed,
+  defineComponent,
+  ref,
+} from 'vue';
+import store from '../store/index';
+import FormErrors from './form/FormErrors.vue';
+import CustomInput from './form/CustomInput.vue';
+import CustomButton from './form/CustomButton.vue';
+import InputModel from '../models/InputModel';
+import ButtonModel from '../models/ButtonModel';
+import MessageModel from '../models/MessageModel';
+
+export default defineComponent({
+  name: 'Login',
+  components: {
+    FormErrors,
+    CustomInput,
+    CustomButton,
+  },
+  setup() {
+    const email = ref('');
+    const password = ref('');
+    const errroMessages = ref<MessageModel[]>([]);
+    const contentLogin = computed(() => store.getters.contentLogin);
+    const customEmailInput = ref<InputModel>({
+      customClass: 'input-email',
+      fieldIcon: 'email',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8211123)
```suggestion
      fieldIcon: "email",
```